presto-docs/src/main/sphinx/release/release-0.286.rst

```diff
@@ -0,0 +1,89 @@
+* Add an feature config property `khyperloglog-agg-group-limit` to set the maximum number of groups `khyperloglog_agg` can have. It will fail the query when the limit is exceeded. The default is 0 which means no limit.
+* Add an feature config property `limit-khyperloglog-agg-group-number-enabled` to control whether to limit the number of groups for aggregation functions using KHyperLogLog state.
+* Add an optimization to optimize queries which has equivalence check filter or constant assignments. Controlled by session property `rewrite_expression_with_constant_expression` and default to enabled.
+* Add an optimization to rewrite left join with array contains in join condition to equi join, the optimization is controlled by session property `rewrite_left_join_array_contains_to_equi_join `.
+* Add client option to disable redirects.
+* Add simple configurations to prestodb/presto docker image and make the image easier to use, including ``config.properties`` and ``jvm.config``.
+* Add support for running on Linux and M1/M2 macOS aarch64 (ARM64).
+* Add validation to check that next URI host and port does not change during query execution.
+* Added 1 folder(ecosystem) and 1 file in sphinx.
+* Added 1 line in sphinx/index.rst.
+* Added CTE hash partition count session property.
+* Added Helm Charts to Installation documentation.
+* Remove redundant sort columns from Plan nodes if a unique constraint can be identified for a prefix of the ordering list.
+* A new changelog table has been added to the Iceberg connector.
+* A reservoir_sample aggregation function is now available.
+* Deprecate exchange.async-page-transport-enabled configuration property.
+* Pass extra credentials such as CAT tokens for definer mode in views.
+
+Iceberg Connector Changes
+_________________________
+* Upgrade Apache Iceberg to 1.4.3.
+
+SPI Changes
+___________
+* Add support for connectors to return joins  in ``ConnectorPlanOptimizer.optimize``.
+
+Hive Changes
+____________
+* Fix parquet dereference pushdown, which fails when parquet_use_column_names is not enabled before.
+* Fixed CTE Materialization for unsupported hive bucket types.
+* Deprecate hive config ``hive.s3.use-instance-credentials``. This config will be removed in the upcoming release (:issue:`21633`).
+
+Hudi Changes
+____________
+* Upgrade Hudi version to 0.14.0.
+
+Iceberg Changes
+_______________
+* Add Iceberg Filter Pushdown Optimizer Rule for execution with Velox.
+* Add ``iceberg.pushdown-filter-enabled`` config property to Iceberg Connector. This config property will control the behaviour of Filter Pushdown in the iceberg connector.
+* Add register and unregister procedures for Iceberg tables.
+* Add support for applying equality deletes as a join. This can be enabled with the session property ``iceberg.delete_as_join_pushdown_enabled``.
```

Comment:
* Add support for applying equality deletes as a join. This can be configured with the session property ``iceberg.delete_as_join_rewrite_enabled`` (enabled by default). :pr:`21605`